lib/apb.ts

```diff
@@ -210,19 +212,45 @@ export class apb {
     }
   }
 
+  generateParametersForSoclessTask(state_name: string, handle_state_kwargs: Record<string, any>){
+    const parameters : SoclessTaskStepParameters = {
+      "execution_id.$" : "$.execution_id",
+      "artifacts.$" : "$.artifacts",
+      "errors.$" : "$.errors",
+      "results.$" : "$.results",
+      "State_Config" : {
+        "Name" : state_name,
+        "Parameters" : handle_state_kwargs
+      },
+    }
+    return parameters
+  }
+
+  //! Interactions still need helper states ):
```

Comment:
question: I believe this comment is now a lie? Or is it the next PR that makes it a lie?